<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
       <width>812</width>
       <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Расписание</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="price_box">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>104</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>270</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Price</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="tax_rate">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>360</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Tax Rate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_tax_button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Tax</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="results_window">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
      <widget class="QPushButton" name="updGlButton">
          <property name="geometry">
              <rect>
                  <x>20</x>
                  <y>20</y>
                  <width>121</width>
                  <height>23</height>
              </rect>
          </property>
          <property name="text">
              <string>Обновит список групп</string>
          </property>
      </widget>
      <widget class="QPushButton" name="dwnldButton">
    <property name="geometry">
     <rect>
         <x>220</x>
         <y>30</y>
         <width>88</width>
         <height>23</height>
     </rect>
    </property>
          <property name="text">
              <string>Скачать файлы</string>
          </property>
      </widget>
      <widget class="QPushButton" name="parseButton">
          <property name="geometry">
              <rect>
                  <x>410</x>
                  <y>30</y>
                  <width>84</width>
                  <height>23</height>
              </rect>
          </property>
          <property name="text">
              <string>Парсить</string>
          </property>
      </widget>
      <widget class="QLabel" name="label_3">
          <property name="geometry">
              <rect>
                  <x>7</x>
                  <y>59</y>
                  <width>40</width>
                  <height>16</height>
              </rect>
          </property>
          <property name="text">
              <string>Группа:</string>
          </property>
      </widget>
      <widget class="QComboBox" name="groupComboBox">
          <property name="geometry">
              <rect>
                  <x>60</x>
                  <y>60</y>
                  <width>100</width>
                  <height>20</height>
              </rect>
          </property>
          <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
              </sizepolicy>
          </property>
          <property name="minimumSize">
              <size>
                  <width>100</width>
                  <height>0</height>
              </size>
          </property>
          <property name="baseSize">
              <size>
                  <width>100</width>
                  <height>0</height>
              </size>
          </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
        <width>812</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
